docs/pages/actions/apollo/apollo.yaml

```diff
@@ -1826,30 +1830,63 @@ paths:
               examples:
                 example-1:
```

Comment:
you'll need to modify the .mdx file for this to be reflected irrc

Reply:
The .mdx example only shows the request body, right?